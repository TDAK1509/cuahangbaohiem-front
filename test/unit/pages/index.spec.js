import { mount } from "@vue/test-utils";
import HomePage from "@/pages/index.vue";

describe("pages/index.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(HomePage);
    expect(wrapper.element).toMatchSnapshot();
  });
});
