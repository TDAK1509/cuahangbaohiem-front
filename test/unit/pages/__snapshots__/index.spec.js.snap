// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index.vue matches snapshot 1`] = `
<div
  class="home container"
>
  <client-only>
    <vue-carousel
      autoplay="true"
      centermode="true"
      loop="true"
      perpage="1"
    >
      <vue-carousel-slide>
        <figure
          class="home__slide"
        >
          <img
            class="home__img"
            src="~/assets/images/slide-1.png"
          />
        </figure>
      </vue-carousel-slide>
       
      <vue-carousel-slide>
        <figure
          class="home__slide"
        >
          <img
            class="home__img"
            src="~/assets/images/slide-2.png"
          />
        </figure>
      </vue-carousel-slide>
    </vue-carousel>
  </client-only>
</div>
`;
